sdk/apis/core/v1alpha1/logicalcluster_types.go

```diff
@@ -59,13 +59,17 @@ const (
 
 // LogicalClusterPhaseType is the type of the current phase of the logical cluster.
 //
-// +kubebuilder:validation:Enum=Scheduling;Initializing;Ready
+// +kubebuilder:validation:Enum=Scheduling;Initializing;Ready;Unavailable
 type LogicalClusterPhaseType string
 
 const (
 	LogicalClusterPhaseScheduling   LogicalClusterPhaseType = "Scheduling"
 	LogicalClusterPhaseInitializing LogicalClusterPhaseType = "Initializing"
 	LogicalClusterPhaseReady        LogicalClusterPhaseType = "Ready"
+	// LogicalClusterPhaseUnavailable phase is used to indicate that the logical cluster us unavailable to be used.
+	// It will will not be served via front-proxy when in this state.
```

Comment:
would be more precise: what does front-proxy return? I would expect 503.

Reply:
401 as with all not found resources. we would need to change front-proxy to return something else